user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Query_updates.md

```diff
@@ -0,0 +1,67 @@
+---
+uid: Query_updates
+---
+
+# Query updates
+
+When executing a query, you have the option to keep your query result up to date by enabling query updates.
+These updates may add, remove or update rows from the initial result returned by your query.
+
+> [!TIP]
+> Take a look at [query update support](#query-update-support) to see which updates your query can support.
+
+## How to enable updates
+
+GQI allows you to enable or disable updates for each query individually.
+
+In the UI however, you control updates per component under *Settings* > *Data retrieval* > *Update data*.
+This setting will enable updates for all queries executed by that component.
+
+## Query update support
+
+A GQI query can offer various levels of support for updates:
+
+1. It can support updates through [real-time events](#real-time-events)
+1. It can support updates through [notify events](#notification-events)
+1. It might not provide any support for updates.
+
+> [!IMPORTANT]
+> The update support of your query is determined by the **lowest** level of support among the data sources and operators in your query.
+> See [supported data sources](#supported-data-sources) and [supported operators](#supported-operators) for an overview.
+
+### Real-time events
+
+As the name suggests, real-time events occur as soon as a change is detected.
+These events contain incremental updates in the form of added, removed or updated rows and cells.
```

Comment:
Do we need to mention that some data sources do not support additions or deletions?

Reply:
I don't think here is the right place.
I mentioned we don't support delete events yet on the View Adapter page, and also added an asterisk here on the support overview to indicate that there are adapter-specific limitations.